Submitted By:            Douglas R. Reno <renodr at linuxfromscratch dot org>
Date:                    2022-08-22
Initial Package Version: 3.24.1
Upstream Status:         Pending
Origin:                  Upstream (MR 7600)
Description:             Adapts CMake's FindBoost module to understand that
                         Boost-1.80.0 is stable and functional. Without this,
                         packages that use CMake to find boost will return an
                         invalid version error.

diff -Naurp cmake-3.24.1.orig/Modules/FindBoost.cmake cmake-3.24.1/Modules/FindBoost.cmake
--- cmake-3.24.1.orig/Modules/FindBoost.cmake	2022-08-17 11:54:57.000000000 -0500
+++ cmake-3.24.1/Modules/FindBoost.cmake	2022-08-22 10:47:35.142625476 -0500
@@ -1380,7 +1380,7 @@ function(_Boost_COMPONENT_DEPENDENCIES c
       set(_Boost_TIMER_DEPENDENCIES chrono)
       set(_Boost_WAVE_DEPENDENCIES filesystem serialization thread chrono atomic)
       set(_Boost_WSERIALIZATION_DEPENDENCIES serialization)
-      if(Boost_VERSION_STRING VERSION_GREATER_EQUAL 1.80.0 AND NOT Boost_NO_WARN_NEW_VERSIONS)
+      if(Boost_VERSION_STRING VERSION_GREATER_EQUAL 1.81.0 AND NOT Boost_NO_WARN_NEW_VERSIONS)
         message(WARNING "New Boost version may have incorrect or missing dependencies and imported targets")
       endif()
     endif()
@@ -1653,7 +1653,7 @@ else()
   # _Boost_COMPONENT_HEADERS.  See the instructions at the top of
   # _Boost_COMPONENT_DEPENDENCIES.
   set(_Boost_KNOWN_VERSIONS ${Boost_ADDITIONAL_VERSIONS}
-    "1.79.0" "1.79"
+    "1.80.0" "1.80" "1.79.0" "1.79"
     "1.78.0" "1.78" "1.77.0" "1.77" "1.76.0" "1.76" "1.75.0" "1.75" "1.74.0" "1.74"
     "1.73.0" "1.73" "1.72.0" "1.72" "1.71.0" "1.71" "1.70.0" "1.70" "1.69.0" "1.69"
     "1.68.0" "1.68" "1.67.0" "1.67" "1.66.0" "1.66" "1.65.1" "1.65.0" "1.65"
